packages/core/src/components/overlay/_overlay.scss

```diff
@@ -94,4 +99,10 @@ $overlay-background-color: rgba($black, 0.7);
   &:focus {
     outline: none;
   }
+
+  // when rendered inline, we want the overlay to position itself relative to
+  // its parent container, not relative to the whole window.
+  .pt-overlay-inline & {
+    @include position(absolute, 0);
```

Comment:
`position: absolute` should be enough. 
this mixin generates five lines, four of which are the same as line 92.